tests/model_serving/model_runtime/mlserver/conftest.py

```
@@ -0,0 +1,158 @@
+        aws_s3_endpoint=models_s3_bucket_endpoint,
+    ) as secret:
+        yield secret
+
+
+@pytest.fixture
+def response_snapshot(snapshot: Any) -> Any:
+    return snapshot.use_extension(extension_class=JSONSnapshotExtension)
+
+
+@pytest.fixture
+def mlserver_pod_resource(admin_client: DynamicClient, mlserver_inference_service: InferenceService) -> Pod:
```

Pylint (3.3.7)

[convention] Line too long (108/100)

(C0301)